src/Exception/Binary/Loader/ChainAttemptNotLoadableException.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Exception\Binary\Loader;
+
+use Liip\ImagineBundle\Binary\Loader\LoaderInterface;
+
+class ChainAttemptNotLoadableException extends NotLoadableException
```

Comment:
```suggestion
final class ChainAttemptNotLoadableException extends NotLoadableException
```
I think we should make them `final` by default if it is not designed to be extended.

Reply:
i agree, makes it much more explicit to know for backwards compatibility

Reply:
Excellent idea; done.